### PR TITLE
Add wait for confirmation function

### PR DESCRIPTION
**Pull request by jasonpaulos**
Add wait for confirmation function for verifying transaction confirmation.

### CODE REVIEW

**Review by algochoi — COMMENTED** [review]
Nice addition to the JS SDK 🎉 

I'm also curious to whether this should be kept with `algod` or `transactions` (I believe the Python SDK binds this method to the latter).

**Review by ahangsu — COMMENTED** [review]
I don't have general comments here. Just curious, why there are 2 alike implementation for WaitForConfirmation in JS-SDK?

**Review by ahangsu — APPROVED** [review]
LGTM

File: tests/cucumber/steps/steps.js [hunk 1]
```diff
@@ -4242,11 +4242,12 @@ module.exports = function getSteps(options) {
       const fundingResponse = await this.v2Client
         .sendRawTransaction(stxKmd)
         .do();
-      await this.v2Client.statusAfterBlock(sp.firstRound + 2).do();
-      const fundingConfirmation = await this.acl.transactionById(
-        fundingResponse.txId
+      const info = await algosdk.waitForConfirmation(
```
[the remark this answers is not in the document]
Comment (jasonpaulos) [reply tests/cucumber/steps/steps.js:4245]: Yes that's the v1 algod client

**Comment by jasonpaulos** [comment]
My bad, I originally forgot to delete the file in the algod client folder.

File: tests/cucumber/steps/steps.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (ahangsu) [reply tests/cucumber/steps/steps.js:4245]: Seems to be true, see https://github.com/algorand/js-algorand-sdk/blob/develop/tests/cucumber/steps/steps.js#L107

**Review by algochoi — APPROVED** [review]

**Pull request merged**